waveorder/stokes.py

```diff
@@ -0,0 +1,344 @@
+"""
+Overview
+--------
+
+This module collects Stokes- and Mueller-related calculations. 
+
+The functions are organized into three groups:
```

Comment:
Not critical:
For now the forward/inverse/convenience are separated by comments in the source file. How about arranging them in a hierarchy so that from user side it's clear what function group it is:

```py
from waveorder import stokes

stokes.forward.A_matrix(...)

stokes.inverse.s0123_CPL_after_ADR(...)

stokes.utils.mmul(...)
```

Reply:
I'm very open to this. Let's discuss more tomorrow morning.